server/scroll.php

```diff
@@ -69,9 +75,10 @@
   $node->{'Time'} = preg_replace('/<[^>]*>/', '', $node->{'Time'});
   $node->{'Day'} = preg_replace('/<[^>]*>/', '', $node->{'Day'});
   
-  // Remove Spaces so we can use it for a CSS class
+  // Remove special chars in Topic from XML request so we can use it for a CSS class
   $node->{'Topic'} = preg_replace('/\s+/', '', $node->{'Topic'});
   $node->{'Topic'} = preg_replace('/\&/', 'and', $node->{'Topic'});
+  $node->{'Topic'} = preg_replace('/\@/', '', $node->{'Topic'});
```

Comment:
@carlynlee did a little digging, turns out we need to replace the `@` in the original Topic name we get from the XML as well. Its due to the lookup that takes place here: https://github.com/socallinuxexpo/scale-signs/blob/2c23457c3f9656ac391a9be722490ba4d287c683/server/scroll.php#L257

It all looks good for me now :slightly_smiling_face: 
![ss-202403061709684396](https://github.com/socallinuxexpo/scale-signs/assets/30531572/19d80422-4f0b-4a8f-a6d9-7a4bda8cc300)

I pushed these changes directly on your branch so just pull them down with: `git pull -r origin shorten_topics_tracklist` and you should these my additions

Reply:
Thanks @sarcasticadmin ! I pulled your changes and it seems to work on my end too:
<img width="1499" alt="Screenshot 2024-03-06 at 9 43 21 AM" src="https://github.com/socallinuxexpo/scale-signs/assets/916785/a2ffc874-a9f6-4467-859d-47bf7369ae07">